llvm/test/Transforms/Scalarizer/frexp.ll

```diff
@@ -0,0 +1,67 @@
+; RUN: opt %s -passes='function(scalarizer<load-store>)' -S | FileCheck %s
```

Comment:
Please generate this using update_test_checks.py.

Reply:
Could you please also add a test where the struct is returned from the function (not using extractelement)?

Reply:
add a `test_vector_frexp_void`. It only does the calls.